int16 X_Coord
int16 Y_Coord
uint16 Z_Coord
